server/app/views/admin/programs/ProgramImageView.java

```diff
@@ -118,6 +129,24 @@ public Content render(Http.Request request, ProgramDefinition programDefinition)
     return layout.renderCentered(htmlBundle);
   }
 
+  private DivTag createBreadcrumbs(ProgramDefinition program) {
+    ImmutableList<BreadcrumbItem> breadcrumbItems =
+        ImmutableList.of(
+            BreadcrumbItem.create(
+                "Edit Programs",
```

Comment:
**thought (if-minor, non-blocking):** Eventually, it would be good if the titles of the breadcrumbs are standardized (via constants or similar) for a consistent UX. Wouldn't want someone calling this breadcrumb something slightly different in another view.